### PR TITLE
adding some cafe brands

### PR DESCRIPTION
**Pull request by waso99**
adding some cafe brands

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
L634
L651

those places misssing comma

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -571,24 +571,11 @@
...
-          "bh",
-          "eg",
-          "kw",
-          "om",
-          "qa",
-          "sa",
-          "tr",
-          "us"
-        ],
-        "exclude": ["cn"]
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:590]: exclude cn can be removed

File: data/brands/amenity/cafe.json [hunk 2]
```diff
@@ -599,6 +586,109 @@
...
+        "amenity": "cafe",
+        "brand": "جافا تايم",
+        "brand:ar": "جافا تايم",
+        "brand:en": "Java Time",
+        "brand:wikidata": "Q12204153",
+        "brand:wikipedia": "ar:جافا تايم",
+        "cuisine": "coffee_shop",
+        "name": "جافا تايم",
+        "name:ar": "جافا تايم",
+        "name:en": "Java Time"
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:618]: miss a comma

File: data/brands/amenity/cafe.json [hunk 3]
```diff
@@ -599,6 +586,109 @@
...
+      "tags": {
+        "amenity": "cafe",
+        "brand": "هاف مليون",
+        "brand:ar": "هاف مليون",
+        "brand:en": "Half Million",
+        "brand:wikidata": "Q108537438",
+        "cuisine": "coffee_shop",
+        "name": "هاف مليون",
+        "name:ar": "هاف مليون",
+        "name:en": "Half Million"
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:634]: miss a comma

File: data/brands/amenity/cafe.json [hunk 4]
```diff
@@ -599,6 +586,109 @@
...
+        "amenity": "cafe",
+        "brand": "د.كيف",
+        "brand:ar": "د.كيف",
+        "brand:en": "dr-cafe",
+        "brand:wikidata": "Q105714225",
+        "brand:wikipedia": "ar:د.كيف",
+        "cuisine": "coffee_shop",
+        "name": "د.كيف",
+        "name:ar": "د.كيف",
+        "name:en": "dr-cafe"
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:651]: miss a comma

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/amenity/cafe.json (hunk 4 above)
Comment (waso99) [reply data/brands/amenity/cafe.json:651]: sorry, I will correct it.

**Pull request merged**